src/doc/src/reference/unstable.md

```diff
@@ -338,26 +338,26 @@ This was stabilized in 1.79 in [#13608](https://github.com/rust-lang/cargo/pull/
 ### MSRV-aware resolver
 
 `-Zmsrv-policy` allows access to an MSRV-aware resolver which can be enabled with:
-- `resolver.something-like-precedence` config field
+- `resolver.incompatible-rust-versions` config field
 - `workspace.resolver = "3"` / `package.resolver = "3"`
 - `package.edition = "2024"` (only in workspace root)
 
 The resolver will prefer dependencies with a `package.rust-version` that is the same or older than your project's MSRV.
 Your project's MSRV is determined by taking the lowest `package.rust-version` set among your workspace members.
 If there is none set, your toolchain version will be used with the intent to pick up the version from rustup's `rust-toolchain.toml`, if present.
 
-#### `resolver.something-like-precedence`
+#### `resolver.incompatible-rust-versions`
 * Type: string
-* Default: "something-like-maximum"
-* Environment: `CARGO_RESOLVER_SOMETHING_LIKE_PRECEDENCE`
+* Default: "allow"
+* Environment: `CARGO_RESOLVER_INCOMPATIBLE_RUST_VERSIONS`
 
-Select which policy should be used when resolving dependencies.  Values include
-- `something-like-maximum`: prefer highest compatible versions of a package
-- `something-like-rust-version`: prefer versions of packages compatible with your project's Rust version
+Select how packages with incompatible rust-versions should be resolved.  Values include:
+- `allow`: treat them like any other dependency
```

Comment:
This is also unclear as user doesn't know how other dependency is treated. Are we making it vague for rooms for future behavior changes?

Reply:
I can't think of a reason to be intentionally vague about `allow`, I'm just unsure of what more needs to be described.

Reply:
I think the problem is pronouns and “other”. What is included in “other”? Maybe something like


```suggestion
- `allow`: treat `package.rust-version` incompatible dependency as compatible
- `fallback`: only consider `package.rust-version` incompatible dependency if no other compatible dependency version matched
```

Pronouns are often confusing, and I prefer to use them less.

(Maybe it is just me 😬)

Reply:
To be fully correct, its an "incompatible dependency version" (not just dependency).  This starts to become a mouthful.  I gave it a try by skipping on `package.` and just saying "version".